### PR TITLE
getBalance added

### PR DESCRIPTION
**Pull request by JerzyLa**

### CODE REVIEW

**Review by PatrykLucka — APPROVED** [review]

File: functions/src/getBalance.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    }
+
+    // create JOToken instance
+    const source = fs.readFileSync(require.resolve('./../build/contracts/JOToken.json'))
+    const parsedSource = JSON.parse(source)
+    const JOToken = new web3.eth.Contract(parsedSource.abi, jotokenAddress)
+
+    try {
+        const balance = await JOToken.methods.balanceOf(data.address).call()
+        return {'balance': balance}
```
Comment (PatrykLucka) [inline functions/src/getBalance.ts:21]: it can be replaced with `return { balance }`

File: functions/src/getBalance.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    }
+    catch(error) {
+        console.log('error: ', error)
```
Comment (PatrykLucka) [inline functions/src/getBalance.ts:24]: errors is better to log with `console.error()` function

File: functions/src/getBalance.ts (hunk 1 above)
Comment (JerzyLa) [reply functions/src/getBalance.ts:21]: will do in next PR

File: functions/src/getBalance.ts (hunk 2 above)
Comment (JerzyLa) [reply functions/src/getBalance.ts:24]: will do in next PR

**Pull request merged**